src/views/Log.vue

```diff
@@ -0,0 +1,234 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<style>
+.v-application--is-ltr .v-text-field__prefix {
+  text-align: right;
+  padding-right: 0px;
+  color: blue;
+}
+</style>
+<template>
+  <div class="h-100">
+    <v-form>
+      <v-container>
+        <v-row justify="start">
+          <v-col cols="12" md="4" >
+            <v-text-field
```

Comment:
This row can overflow for narrow displays and or long workflow IDs. It would probably be better to allow the form to spread over multiple lines is necessary, but this isn't release blocking, if we can come up with a quick fix, great, if not, no biggie, we can always address this in UI 1.4.1 or whatnot.

<img width="924" alt="Screenshot 2023-01-13 at 09 46 14" src="https://user-images.githubusercontent.com/16705946/212290428-2cbba947-84ce-4715-a213-25e02b2f475f.png">